src/openrct2/windows/top_toolbar.c

```diff
@@ -3090,6 +3094,8 @@ void top_toolbar_init_view_menu(rct_window* w, rct_widget* widget) {
 		dropdown_set_checked(11, true);
 	if (mainViewport->flags & VIEWPORT_FLAG_PATH_HEIGHTS)
 		dropdown_set_checked(12, true);
+	if (mainViewport->flags & VIEWPORT_FLAG_PAINT_CLIP_TO_HEIGHT)
+		dropdown_set_checked(14, true);
```

Comment:
`DDIDX_VIEW_CLIPPING` instead of `14` (we're trying to get rid of all magic numbers, so I'd like to avoid adding in new ones)